java-client/src/main/java/com/wavefront/integrations/WavefrontDirectSender.java

```diff
@@ -99,6 +149,60 @@ static String pointToString(String name, double value, @Nullable Long timestamp,
     }
     sb.append("source=").append(quote).append(escapeQuotes(source)).append(quote);
 
+    sb.append(pointTagsToString(pointTags));
+    return sb.toString();
+  }
+
+  @Nullable
+  static String histogramToString(HistogramGranularity histogramGranularity, @Nullable Long timestamp,
```

Comment:
Let's create a HistogramConstants.java under https://github.com/wavefrontHQ/java/tree/b9c5258ff7bf729cf4af82af5ca55c9850f49e16/java-lib/src/main/java/com/wavefront/common and put this logic there and re-use here and Wavefront.java.

Have the common code throw an IllegalArgumentException but catch and log here and use as is within Wavefront.java.